release-notes/8.0/supported-os.json

```diff
@@ -0,0 +1,48 @@
+{
+  "channel-version": "8.0",
+  "last-updated": "2024-02-28",
+  "glibc": [
```

Comment:
https://github.com/Falco20019/core/blob/extract-release-json/release-notes/8.0/supported-os.md#libc-compatibility has a lot shorter list than .NET 7 and is also dropping the architectures. Does that mean that there is no limitation or does it mean "the same architectures as above"?